.github/workflows/ci.yml

```diff
@@ -0,0 +1,16 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+  # Every Sunday at 12:20pm UTC
+  schedule:
+    - cron: '20 12 * * 7'
+
+jobs:
+  ci:
+    name: CI
+    # Only run cron on the silverstripe account
+    if: (github.event_name == 'schedule' && startsWith(github.repository, 'silverstripe/')) || (github.event_name != 'schedule')
```

Comment:
Is this right for the symbiote projects?

Reply:
A thoughtful idea, though I'm not entirely sure that would work, the intention here is to ensure that cron jobs do not run on forked repos (including the creative-commoners account). I think github.repository_owner would point to the account of the forked repo on push events.

Reply:
That's a good idea, though I've already pushed a [whole lot](https://github.com/silverstripe/gha-ci/issues/11) of pull-requests :-) Might take a bit too much work to update them all

Reply:
Yeah, indeed - I don't expect you to change them all 😅

Reply:
I've added it as a checkbox to the "Round2 PRs" on the parent issue.  We'll need another round of PR's to remove the .travis.yml files, so we can bulk update the workflow files then.  I wouldn't be surprised if we come up with something else we want to bulk update at the same time.